### ISSUE TITLE
JOSS REVIEW - Performance claims in the manuscript not backed

### ISSUE THREAD

**Issue by Robaina** [biocircuits/bioscrape#156]
Hi!

I'm reviewing this [JOSS submission](https://github.com/openjournals/joss-reviews/issues/5057). I noticed that the authors claim in the manuscript that bioscrape runs faster than the current alternatives:

```
"Some popular software packages that do somewhat similar tasks as the bioscrape package are MATLAB’s SimBiology toolbox (MATLAB, 2016), Stochpy (Maarleveld, 2013), COPASI (Hoops et al., 2006), and Tellurium (Medley et al., 2018). However, the bioscrape package is faster [...] "
```

I think that such a claim should be backed by some sort of performance benchmark test among competing tools.

I'm raising the issue here to keep track of the comments.

**Comment by ayush9pandey** [comment]
Thanks! Fix started with the [`benchmarking`](https://github.com/biocircuits/bioscrape/tree/benchmarking) branch.

**Comment by ayush9pandey** [comment]
Fixed and merged to master with #160
